scss/components/highlight-banner/_highlight-banner.scss

```diff
@@ -9,15 +9,27 @@
 // Grab our mixins.
 @import "utilities/utilities";
 
-.highlight-card {
-  box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.1);
-  padding: 30px 30px 20px;
-
-  p {
-    color: color(ash);
-    letter-spacing: .5px;
-    line-height: 2em;
-    margin-bottom: .5em;
+.highlight-banner__container {
+  .front & {
+    top: -35px;
+  }
+
+  .highlight-cards {
+    .highlight-card {
+      box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.1);
+      padding: 30px 30px 20px;
+
+      .front & {
```

Comment:
Are you sure you want all highlight cards on the home page for all of time to have a margin bottom of this? What if others get added or moved over time?